new-client/src/plugins/LayerSwitcher/components/BackgroundSwitcher.js

```diff
@@ -63,12 +66,14 @@ class BackgroundSwitcher extends React.PureComponent {
       const osmSource = new OSM({
         reprojectionErrorThreshold: 5,
       });
-      this.osmLayer = new TileLayer({
-        visible: false,
-        source: osmSource,
-        zIndex: -1,
-      });
-      this.props.map.addLayer(this.osmLayer);
+      this.osmLayer.push(
+        new TileLayer({
+          visible: false,
+          source: osmSource,
+          zIndex: -1,
+        })
+      );
+      this.props.map.addLayer(this.osmLayer[0]);
```

Comment:
There might be a better way but the problem is that the osmlayer is created inside componentdidmount but we need to send pass it before it is created, hence using a list. Basically, the LayerItem for the OSM layer is created before the osmlayer is created in componentdidmount but we need to pass something so we can reference the layer later. 

Now that I think about it I guess it would be better just to init the layer in the constructor and only wait with putting it on the map until componentdidmount. I will make this change. The idea was not to make it hold more than one layer, just to have a value to pass so we can access the osmlayer after it has been created.

Reply:
Moved the init of osmLayer to constructor instead so we do not have to use a list.